en/collaborative-work/export/customexports.md

```diff
@@ -138,6 +138,7 @@ JabRef provides the following set of formatters:
 * `RisKeywords` : to be documented.
 * `RisMonth` : to be documented.
 * `RTFChars` : replaces TeX-specific special characters \(e.g. {\^a} or {\"{o}}\) with their RTF representations, and translates LaTeX commands \emph, \textit, \textbf into RTF equivalents.
+* `ShortMonth` : formats the month field to use abbreviated format.
```

Comment:
Could you add one or two examples? A short month could be APR, apr, Apr, aprl, 04, #apr#, ... It should be clear to the user which abbrevation is taken.